packages/paste-website/src/components/site-wrapper/sidebar/SidebarAnchor.tsx

```diff
@@ -2,28 +2,47 @@ import * as React from 'react';
 import {styled, css} from '@twilio-paste/styling-library';
 import {Link} from 'gatsby';
 
+import {SidebarItem} from './SidebarItem';
+
 interface SidebarAnchorProps {
   children: React.ReactNode;
-  nested?: boolean;
+  level: 0 | 1 | 2;
   to: string;
   onClick?: () => void;
 }
 
+const levelPaddingMap = {
```

Comment:
The padding for the nav item levels should be what's in the table 👇 That's my bad for not redlining for the handoff, can definitely do that next time around.
Level 1 | Level 2 & 3
--- | ---
![image](https://user-images.githubusercontent.com/23108291/149260351-eccd5a62-b170-42c9-9d81-18ca1bfe630d.png) | ![image](https://user-images.githubusercontent.com/23108291/149260368-a0eeac4f-6767-4d61-a2fa-94024d047d89.png)

Reply:
+1 to the focus ring spanning the entire width of the sidebar if that's doable.